src/common/providers/identity.ts

```diff
@@ -1072,9 +866,9 @@ function wrapHandler(
         err = new HttpsError('internal', 'An unexpected error occurred.');
       }
 
-      res.status(err.code);
-      res.setHeader('Content-Type', 'application/json');
-      res.send({ error: err.toJson() });
+      const { status } = err.httpErrorCode;
+      const body = { error: err.toJSON() };
```

Comment:
Is there no need to set Content-Type header?

Reply:
woops, fixed